### ISSUE TITLE
Error running to "X_offset = x.gather(dim=-1, index=index).contiguous().view(b, C, H, W, N)".

### ISSUE THREAD

**Issue by yonghua-wu** [RanSuLab/DUNet-retinal-vessel-detection#7]
Error running to "X_offset = x.gather(dim=-1, index=index).contiguous().view(b, C, H, W, N)".
Just like #2.
This is an accidental error.

![image](https://user-images.githubusercontent.com/37923411/147917319-39355a40-dfb2-4f0c-bff3-caaa74e0d287.png)

My environment:
pytorch 1.10.1
cuda 11.3

Dataset is DRIVE

**Comment by wozc1112** [comment]
hi,could you please tell me the version of keras and tensorflow
Thank you very much if I can get your answer
